querybook/webapp/components/QueryEditor/QueryEditor.tsx

```diff
@@ -408,6 +450,37 @@ export const QueryEditor: React.FC<
             [isTokenInTable, matchFunctionWithDefinition, openTableModal]
         );
 
+        const onTextHoverShortDebounce = useMemo(
```

Comment:
Nice! i think it would be useful if you add a useDebouncedFn hook to reduce the boilerplate